Core/include/Acts/Seeding/SeedFinderOrthogonal.ipp

```diff
@@ -353,7 +356,8 @@ void SeedFinderOrthogonal<external_spacepoint_t>::filterCandidates(
     }
     if (!top_valid.empty()) {
       m_config.seedFilter->filterSeeds_2SpFixed(
-          *bottom[b], middle, top_valid, curvatures, impactParameters, Zob, it);
+          *bottom[b], middle, top_valid, curvatures, impactParameters, Zob,
+          numQualitySeeds, numSeeds it);
```

Comment:
missing `,` here between `numSeeds` and `it`?

Reply:
done